recipes/boms/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "boms" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/boms-{{ version }}.tar.gz
```

Probably not very important, but how about also replacing the last occurrence of `boms` in the url with `{{ name }}`?

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
That, at least, was done [here](https://github.com/bioconda/bioconda-recipes/pull/44387/files).

( Or you could go all in on hardcoding the name like it is done [here](https://github.com/bioconda/bioconda-recipes/pull/44391/files) )